chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/contact/PointOfContact.java

```diff
@@ -0,0 +1,87 @@
+package gov.healthit.chpl.domain.contact;
+
+import java.util.HashMap;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlType;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import gov.healthit.chpl.dto.ContactDTO;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.extern.log4j.Log4j2;
+
+@XmlType(namespace = "http://chpl.healthit.gov/listings")
+@XmlAccessorType(XmlAccessType.FIELD)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+@Log4j2
+public class PointOfContact extends Person {
+    private static final long serialVersionUID = -1945872096428814999L;
+
+    /**
+     * Database ID of a point of contact.
+     */
+    @XmlElement(required = true)
+    private Long contactId;
+
+    public PointOfContact() {
+        super();
+    }
+
+    public PointOfContact(ContactDTO dto) {
+        super(dto);
+        this.contactId = dto.getId();
+    }
+
+    @Builder
+    public PointOfContact(String fullName, String email, String phoneNumber, String title, Long contactId) {
```

Comment:
Don't need `final` here either, right?

Reply:
I don't see "final" here... am I missing something?

Reply:
Weird. I was sure I saw that there. Sorry!